azkaban-web-server/src/web/js/azkaban/view/flow-execute-dialog.js

```diff
@@ -496,18 +498,14 @@ var disableFinishedJobs = function(data) {
       node.status = "READY";
       node.disabled = true;
     }
-    else if (node.status == "SUCCEEDED" || node.status=="RUNNING") {
+    else if (node.status == "SUCCEEDED" || node.missing) {
       node.disabled = true;
     }
-    else if (node.status == "CANCELLED") {
```

Comment:
The cancelled nodes will be still enabled after this unsuccessful execution. It makes sense to me that we should keep the peach color when preparing a new execution.